lib/usd/ui/layerEditor/batchSaveLayersUIDelegate.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include <mayaUsd/nodes/layerManager.h>
 #include <mayaUsd/utils/utilSerialization.h>
+#include <mayaUsd/base/tokens.h>
```

Comment:
I have a feeling this will break the clang-format check since the files are to be listed alphabetically.